FE-102/fe-102-browse-page.html

```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../FE-101/main.css" />
+    <link rel="stylesheet" href="browse-page.css" />
+    <title>Clacks & Thocc - Browse</title>
+  </head>
+  <body>
+    <nav class="nav">
+      <a class="title-logo" href="../FE-101/fe101-article-page.html">
+        Clacks&
+        <span>Thocc</span>
+      </a>
+      <img
+        class="nav-logo"
+        src="../FE-101/images/keyboard-logo.png"
+        alt="colored keyboard logo" />
+      <ul>
+        <li><a href="../FE-101/fe101-article-page.html">Home</a></li>
+        <li>
+          <a
+            href="https://www.codeandtheory.com/location/manila"
+            target="_blank">
+            About Us
+          </a>
+        </li>
+      </ul>
+    </nav>
+    <section class="grid-container">
+      <div class="opening-section">
+        <header>
+          <h1>Lorem Ipsum Dolor sit amet</h1>
+        </header>
+        <div class="featured-articles">
+          <div class="feature-article-card">
+            <a href="../FE-102/fe-102-detail-page-sample1.html">
+              <div class="img-box">
+                <img
+                  class="feature-article-img"
+                  src="../FE-101/images/featureImage-SizeDown.jpeg"
+                  alt="two keyboard sizes" />
+              </div>
+              <h2>Down-sizing your keyboard: should you even?</h2>
+            </a>
+          </div>
+          <div class="feature-article-card">
+            <a href="../FE-102/fe-102-detail-page-sample2.html">
+              <div class="img-box">
+                <img
+                  class="feature-article-img"
+                  src="../FE-101/images/featureImage-whiteKeeb.jpeg"
+                  alt="White Keyboard and clean desk" />
+              </div>
+              <h2>How to start a Clean Setup. Coffee not included.</h2>
+            </a>
+          </div>
+          <div class="feature-article-card">
+            <a href="../FE-102/fe-102-detail-page-sample3.html">
+              <div class="img-box">
+                <img
+                  class="feature-article-img"
+                  src="../FE-101/images/featureImage-Typing2.jpeg"
+                  alt="typing on a keyboard" />
+              </div>
+              <h2>Typing adjustment and how to make it easier</h2>
+            </a>
+          </div>
+        </div>
+      </div>
+      <section class="articles-container">
+        <span class="articles-container-header">
+          == Continue exploring below ==
+        </span>
```

Comment:
Let's use a `heading` tag for header elements. Keep in mind the hierarchy order. `h1` should go first, then `h2`, and so on.

Please also check for other instances of this.